src/components/v2/V2Create/forms/RulesForm/index.tsx

```diff
@@ -20,52 +20,35 @@ import {
   DEFAULT_BALLOT_STRATEGY,
 } from 'constants/v2/ballotStrategies'
 import FormItemLabel from '../../FormItemLabel'
-
-type RulesFormFields = {
-  pausePay: boolean
-  allowMinting: boolean
-  ballot: string
-}
+import { getBallotStrategyByAddress } from 'constants/v2/ballotStrategies/getBallotStrategiesByAddress'
 
 export default function RulesForm({ onFinish }: { onFinish: VoidFunction }) {
   const { theme } = useContext(ThemeContext)
 
-  const [form] = Form.useForm<RulesFormFields>()
   const dispatch = useAppDispatch()
   const { fundingCycleMetadata, fundingCycleData } = useAppSelector(
     state => state.editingV2Project,
   )
 
   const [showMintingWarning, setShowMintingWarning] = useState<boolean>(false)
   const [ballotStrategy, setBallotStrategy] = useState<BallotStrategy>(
-    DEFAULT_BALLOT_STRATEGY,
+    getBallotStrategyByAddress(
+      fundingCycleData?.ballot ?? DEFAULT_BALLOT_STRATEGY.address,
+    ),
   )
-
-  const onFormSaved = useCallback(
-    (fields: RulesFormFields) => {
-      dispatch(editingV2ProjectActions.setPausePay(fields.pausePay))
-      dispatch(editingV2ProjectActions.setAllowMinting(fields.allowMinting))
-      dispatch(editingV2ProjectActions.setBallot(ballotStrategy.address))
-      onFinish?.()
-    },
-    [dispatch, onFinish, ballotStrategy],
+  const [pausePay, setPausePay] = useState<boolean>(
```

Comment:
Totally bail on using the `form`. It's a massive pain to get working, there's no need for it here anyway.

Reply:
Yeah lol don't blame you I've been tempted. Good call here for sure